starknet_devnet/util.py

```diff
@@ -241,3 +242,21 @@ def generate_state_update(previous_state: CarriedState, current_state: CarriedSt
             "storage_diffs": storage_diffs
         }
     }
+
+class JsonErrorHandler:
+    """
+    Error handler that outputs errors in JSON format.
+    """
+
+    def __init__(self, error_message, error_code):
+        self.error_code = error_code
+        self.error_message = error_message
+
+    def handle(self):
```

Comment:
Do you think there is benefit from having this as a class rather than a simple function that accepts the error_message and error_code?

Reply:
So, now, the whole `JsonErrorResponse` class is now removed. https://github.com/Shard-Labs/starknet-devnet/pull/81/commits/ac86499da19eb8d3d21b35c52cd0980efd1be8fb